ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2790,6 +2790,35 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-ibmcloud-ipi
+- as: ibmcloud-ipi-ovn-ipsec-f28
```

Comment:
@gpei is the `f28` correct?

Reply:
It basically depends on what frequency the feature owner willing to run this test in our CI, to me, f28 sounds like enough for us to test this in 4.14.